android/src/main/java/com/swmansion/reanimated/ReanimatedModule.java

```diff
@@ -92,7 +92,7 @@ public String getName() {
 
   private NodesManager getNodesManager() {
     if (mNodesManager == null) {
-      mNodesManager = new NodesManager(getReactApplicationContext());
+      mNodesManager = NodesManager.createNodesManager(getReactApplicationContext());
```

Comment:
if the node manager is used as a singleton maybe we should do it in the `createNodesManager()` method?

Reply:
Yup I agree. Optimally I would make createNodesManager private,, or move that logic to constructor that will be private and have just `getInstance()` method to retrieve it and create it when it is not yet initialised